matrix-sdk-android/src/main/java/org/matrix/android/sdk/userstories/Us000InitMatrix.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("unused")
+
+package org.matrix.android.sdk.userstories
+
+/**
+ * ### Title
+ * Init a Matrix object
+ *
+ * ### Required APIs:
+ * - [org.matrix.android.sdk.api.Matrix] constructor
+ */
+object Us000InitMatrix
```

Comment:
I have 2 questions:
- do the user stories really need to be `object`? I think simple `class` could work and avoid loading of non used US at runtime.
- Am I missing something or the US are not implemented yet? If not implemented, would't be better to make it `internal` or at least mention it in the doc?